tempus_fugit_minecraft/shaders.py

```diff
@@ -97,6 +97,7 @@ def adjust_ambient_light(self, red, green, blue):
         @Param blue a number that specifies the intensity of blue light
         @return returns the c_float vector that corresponds to the ambient
         @see [Issue#12](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/12)
+        @see [Issue#18](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/18)
```

Comment:
issue #18 contain 
Creation of Sun object
Creation of Moon object
I suggest editing that while will not do it